universal-login-web3/lib/index.ts

```diff
@@ -32,6 +33,7 @@ const WALLET = {
 export class ULWeb3Provider implements Provider {
   private sdk: UniversalLoginSDK;
   private walletService: WalletService;
+  private uiController: UIController;
```

Comment:
I think it is time to extract `ULWeb3Provider` and `JsonRPCRequest`, `JsonRPCResponse`, `Callback<ResultType>` to seperate files.